app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] **[OPTIONAL]** Here too  `belongs_to :user` is supposed to be `belongs_to :author, class_name: 'User'`.